api/controllers/ApplicationTypeController.js

```diff
@@ -42,72 +51,102 @@ module.exports = {
+                        req.session.appSubmittedStatus = false;
+                        req.session.email_sent = false;
+
+                        return res.view(serviceSelectorView, {
+                            application_id: 0,
+                            userServiceURL:
+                                sails.config.customURLs.userServiceURL,
+                            error_report: false,
+                            changing: false,
+                            form_values: false,
+                            submit_status: req.session.appSubmittedStatus,
+                            current_uri: req.originalUrl,
+                            user_data: userData,
+                            back_link: req.session.startBackLink,
+                            //disableStandardServiceSection: disableStandardServiceSection
+                        });
                     });
-                  });
                 });
             });
 
-        }else{
-            req.session.appId = false; // reset the appId so a new session is used
-            // set initial submit status to false, meaning it application has not yet been submitted
-            req.session.appSubmittedStatus = false;
+        }
+
+        req.session.appId = false; // reset the appId so a new session is used
+        // set initial submit status to false, meaning it application has not yet been submitted
+        req.session.appSubmittedStatus = false;
+
+        return res.view('applicationForms/applicationType.ejs', {
+            application_id: 0,
+            userServiceURL: sails.config.customURLs.userServiceURL,
+            error_report: false,
+            changing: false,
+            form_values: false,
+            submit_status: req.session.appSubmittedStatus,
+            current_uri: req.originalUrl,
+            user_data: HelperService.getUserData(req,res),
+            back_link: req.session.startBackLink,
+            //disableStandardServiceSection: disableStandardServiceSection
+        });
+
+    },
 
-            return res.view('applicationForms/applicationType.ejs', {
-                application_id: 0,
-                userServiceURL: sails.config.customURLs.userServiceURL,
-                error_report: false,
-                changing: false,
-                form_values: false,
-                submit_status: req.session.appSubmittedStatus,
-                current_uri: req.originalUrl,
-                user_data: HelperService.getUserData(req,res),
+    handleServiceChoice(req, res) {
+        const {'choose-a-service': chosenService} = req.body;
+        const userLoggedIn = HelperService.LoggedInStatus(req);
+
+        if(!userLoggedIn) {
+            sails.log.error('User is not logged in');
+            return res.view('404');
+        }
+
+        const servicePages = {
+            eApostille: '/new-application?app_type_group=4',
+            standard: '/new-application?app_type_group=1',
```

Comment:
There is the drop off service as well which would be app_type_group=3

Reply:
At the moment we haven't added the drop-off service to the new applications screen. I think it's something we will add after the MVP but for now it's just the three services.

<img width="634" alt="Screenshot 2021-10-04 at 14 41 51" src="https://user-images.githubusercontent.com/1377253/135862007-81ab5848-bf37-4704-8782-6447d2c8d6c6.png">